new_samples/workflows/hello_world_workflow.go

```diff
@@ -0,0 +1,43 @@
+package workflows
+
+import (
+	"context"
+	"fmt"
+	"go.uber.org/cadence/activity"
+	"go.uber.org/cadence/workflow"
+	"go.uber.org/zap"
+	"time"
+)
+
+type sampleInput struct {
+	Message string `json:"message"`
+}
+
+// HelloWorldWorkflow greets the caller.
+func HelloWorldWorkflow(ctx workflow.Context, input sampleInput) error {
+	ao := workflow.ActivityOptions{
+		StartToCloseTimeout:    time.Minute,
+		ScheduleToStartTimeout: time.Minute,
+	}
+	ctx = workflow.WithActivityOptions(ctx, ao)
+
+	logger := workflow.GetLogger(ctx)
+	logger.Info("HelloWorldWorkflow started")
+
+	var greetingMsg string
```

Comment:
I kinda prefer returning a value in samples because then you can see it in the UI / use `GetWorkflow(..).Get(.., &out)` to read it in other code.

But... it's also just not actually necessary 🤔.  maybe keeping things simple is better for a "hello world"?

Reply:
not a problem, let's do that. I'll make this workflow return this greetingMsg.